### PR TITLE
Make Zazu more accessible.

### LINKED ISSUE #35: Voice over accessibility

### PR DESCRIPTION
**Pull request by bayleedev**
Would fix #35

### CODE REVIEW

File: app/background.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ app.on('ready', function () {
     width: 600,
     height: 400,
     maxHeight: 400,
-    show: debug,
+    show: false,
```
Comment (bayleedev) [inline app/background.js:72]: Changing this to `false` allows me to use debug easier.

File: app/components/result.js [hunk 2]
```diff
@@ -46,6 +51,8 @@ const Result = React.createClass({
         onClick: this.click,
         className: active ? 'active' : 'inactive',
         ref: this.setReference,
+        tabIndex: 0,
```
Comment (bayleedev) [inline app/components/result.js:54]: Tabindex 0 allows non anchor tags to be traversed using tab in dom order.

File: app/components/search.js [hunk 3]
```diff
@@ -20,10 +20,6 @@ const Search = React.createClass({
     this.focus()
   },
 
-  componentDidUpdate () {
-    this.focus()
```
Comment (bayleedev) [inline app/components/search.js:24]: We don't focus as often, which could cause unexpected behavior.

File: features/step_definitions/myStepDefinitions.js [hunk 4]
```diff
@@ -221,6 +225,14 @@ module.exports = function () {
     })
   })
 
+  this.Then(/^I have no accessibility warnings$/, function () {
+    return this.accessibility().then((response) => {
+      if (response.results.length !== 0) {
+        throw new Error('You have accessibility issues')
```
Comment (bayleedev) [inline features/step_definitions/myStepDefinitions.js:231]: This doesn't do a great job at communicating what the errors/warnings are.

**Comment by bayleedev** [comment]
The issue we ran into last night was that Zazu won't autofocus correctly when opened.

File: app/components/search.js (hunk 3 above)
Comment (bayleedev) [reply app/components/search.js:24]: This is now conditional on the query value being empty.

**Comment by afaur** [comment]
Open zazu
Type something that yields multiple results
Select one of the results

Open zazu, 
Type something that yields multiple results
Item at same position as the last item you chose gets highlighted in a lighter color.
last_index_selected_shown_in_new_search

**Comment by bayleedev** [comment]
@afaur I believe the lighter color is the mouse hover. Can you verify?

**Pull request merged**